### PR TITLE
Admin - Test - List Daily Menu

### PR DESCRIPTION
**Pull request by hungbkdn**

### CODE REVIEW

File: tests/Browser/DailyMenu/ListTest.php [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            ->assertPathIs('/daily-menus');
+        });
+    }
+
+    /**
+     * A Dusk test count object in list=10.
+     *
+     * @return void
+     */
+    public function testObject10()
```
Comment (ghost) [inline tests/Browser/DailyMenu/ListTest.php]: what's testObject10? I don't understand it.

File: tests/Browser/DailyMenu/ListTest.php [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            $this->assertTrue(count($objectCount)==11);
+        });
+    }
+
+    /**
+     * A Dusk test count object in list=11.
+     *
+     * @return void
+     */
+    public function testObject11()
```
Comment (ghost) [inline tests/Browser/DailyMenu/ListTest.php]: what't testObject11?

**Review by ghost — CHANGES_REQUESTED** [review]
check comment

**Comment by hungbkdn** [comment]
I have updated, please check it again.
Thank you so much!

File: tests/Browser/AdminListDailyMenuTest.php [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        $category = factory(Category::class, 1)->create();
+        $foods = factory(Food::class, 10)->create(['category_id' => 1]);
+        $dailyMenus = factory(DailyMenu::class, 10)->create([
+            'food_id' => 1
+        ]);
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(1)
+            ->visit('/daily-menus')
+            ->resize(1920, 2000);
+            $objectCount = $browser->elements('.table tr');
```
Comment (ghost) [inline tests/Browser/AdminListDailyMenuTest.php]: You can check with element `.table tbody tr` to get correct data record.

File: tests/Browser/AdminListDailyMenuTest.php [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    }
+
+    /**
+     * @group dailymenu
+     *
+     * A Dusk test count object in list=10.
+     *
+     * @return void
+     */
+    public function testPaginationWith10Rows()
```
Comment (ghost) [inline tests/Browser/AdminListDailyMenuTest.php]: You want to test list has one page right?. If correct, you should be change function name to `testHasOnePage`.

File: tests/Browser/AdminListDailyMenuTest.php [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+    {
+        $category = factory(Category::class, 1)->create();
+        $foods = factory(Food::class, 10)->create(['category_id' => 1]);
```
Comment (at-vietpham) [inline tests/Browser/AdminListDailyMenuTest.php]: Use `factory(Food::class, 10)` is not good. You create 10 record -> count -> compare with 10. What happen when limit per page changed?
For example:
`ITEMS_PER_PAGE` in model `Food` changed to 5. So on one page, we have 5 records. You create 10 records but in list display only 5. Your test fails.

File: tests/Browser/AdminListDailyMenuTest.php [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+    }
+
+    /**
+     * @group dailymenu
+     *
+     * A Dusk test count object in list=11.
+     *
+     * @return void
+     */
+    public function testPaginationMorethan10Rows()
```
Comment (ghost) [inline tests/Browser/AdminListDailyMenuTest.php]: If you want to test case list has many pages. Function name should be `testPagination`

File: tests/Browser/AdminListDailyMenuTest.php [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+        $category = factory(Category::class, 1)->create();
+        $foods = factory(Food::class, 11)->create(['category_id' => 1]);
+        $dailyMenus = factory(DailyMenu::class, 11)->create([
+            'food_id' => 1
+        ]);
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(1)
+            ->visit('/daily-menus')
+            ->resize(1920, 2000);
+            $objectCount = $browser->elements('.table tr');
```
Comment (ghost) [inline tests/Browser/AdminListDailyMenuTest.php]: You can check with element .table tbody tr to get correct data record.

**Review by ghost — COMMENTED** [review]
Please check my comment. You need to add case test when list empty.

**Comment by ghost** [comment]
LGTM 😄

**Review by ghost — APPROVED** [review]
LGTM
